### PR TITLE
Fixed problem from previous patch ;) #619

### PR DESCRIPTION
**Pull request by ivandrofly**
Here @niksedk, sorry for that ;)

### CODE REVIEW

**Comment by ivandrofly** [comment]
AppVeyor fails 'cause patch didn't finished building up

File: src/Forms/RemoveTextFromHearImpaired.cs [hunk 1]
```diff
@@ -165,7 +165,7 @@ public int RemoveTextFromHearImpaired()
                 if (item.Checked)
                 {
                     Paragraph p = (Paragraph)item.Tag;
-                    string newText = columnHeaderAfter.Text;
+                    string newText = item.SubItems[3].Text.Replace(Configuration.Settings.General.ListViewLineSeparatorString, Environment.NewLine);
```
Comment (niksedk) [inline src/Forms/RemoveTextFromHearImpaired.cs]: Very close :)
We're not a 100% sure what the ListViewLineSeparator is... could be space...

Comment (ivandrofly) [inline src/Forms/RemoveTextFromHearImpaired.cs]: yeah... I think neither "<br />" is the right solution

**Comment by ivandrofly** [comment]
I think something can be done...

**Pull request merged**